cli/src/main.rs

```diff
@@ -29,9 +29,20 @@ pub enum DIDKit {
         #[structopt(flatten)]
         key: KeyArg,
     },
-    /// Output a DID for a given JWK and DID method name or pattern.
+    /// Output a DID for a given JWK according to the provided DID method name or pattern
+    ///
+    /// Deterministically generate a DID from a public key JWK, for a DID method
+    /// that support this functionality.
+    ///
+    /// The DID method to use may be provided in the `method-pattern` argument. The default is
+    /// `did:key`.
```

Comment:
Maybe I would just put `key` as it is what should be passed in the CLI, and put in parentheses that it correspond to `did:key`? No strong opinion.

Reply:
Good idea; updated in e697c793567fa63bda751230fee480fa5fe7b7f5